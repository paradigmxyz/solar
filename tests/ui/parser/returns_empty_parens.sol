function f() returns() {}
//~^ ERROR: expected one of `function`, `mapping`, elementary type name, or path, found `)`
