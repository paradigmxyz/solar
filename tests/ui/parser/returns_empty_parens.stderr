error: expected one of `function`, `mapping`, elementary type name, or path, found `)`
  --> ROOT/tests/ui/parser/returns_empty_parens.sol:LL:CC
   |
LL | function f() returns() {}
   |                      ^ expected one of `function`, `mapping`, elementary type name, or path
   |

error: aborting due to 1 previous error

